fitbenchmarking/fitbenchmarking
# 300 table generation

#### Description of Work

Fixes #300 

Converts hard coded tables into pandas tables.

#### Testing Instructions

1. Check tables give the same results as before.
2. Check tables have right colour scales.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
